dev/tools/localization/gen_l10n.dart

```diff
@@ -459,7 +496,7 @@ class LocalizationsGenerator {
 
   static RegExp arbFilenameLocaleRE = RegExp(r'^[^_]*_(\w+)\.arb$');
   static RegExp arbFilenameRE = RegExp(r'(\w+)\.arb$');
-  static RegExp pluralValueRE = RegExp(r'^\s*\{[\w\s,]*,\s*plural\s*,');
+  static RegExp pluralValueRE = RegExp(r'^\s*\{[\w\s]*,\s*plural\s*,[\s\S]*\}');
```

Comment:
I thought of maybe making this a little more precise to help avoid the plural string formatting issue mentioned before. However, I was wondering if this would somehow make the match too strict? @HansMuller

Reply:
I think this is OK as it is. We'll have to see what failure modes actually crop up in real apps before adding more checking.

Reply:
Sounds good, I'll change it to be what it was before